library/core/src/main/java/com/google/android/exoplayer2/trackselection/TrackSelectionParameters.java

```diff
@@ -222,13 +238,13 @@ public static TrackSelectionParameters getDefaults(Context context) {
    * {@code null} selects the default track, or the first track if there's no default. The default
    * value is {@code null}.
    */
-  @Nullable public final String preferredAudioLanguage;
+  public final String[] preferredAudioLanguage;
```

Comment:
Can you make this an ImmutableList to avoid changes after the Parameter object was created? Same below for the text languages.

Also, could you add an "s" to both parameter names to make it "languages"?